packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.stories.js

```diff
@@ -0,0 +1,20 @@
+// /* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text, number } from "@storybook/addon-knobs";
+
+import SfQuantitySelector from "./SfQuantitySelector.vue";
+
+storiesOf("Atoms|QuantitySelector", module)
+  .addDecorator(withKnobs)
+  .add("Default", () => ({
+    components: { SfQuantitySelector },
+    props: {
+      qty: {
+        default: text("qty (prop)", "0")
+      },
+      stock: {
+        default: number("stock (prop)", 0)
```

Comment:
```suggestion
        default: number("stock", 99, "Props")
```
if you set 0 then you cant use it :)

Reply:
Just a small addition: the [number knob](https://github.com/storybookjs/storybook/tree/master/addons/knobs#number) has the `groupId` as the 4th argument 🤷‍♂ very tricky
number(label, defaultValue, **{}, groupId**);

Reply:
haha sure! for text is `text(label, defaultValue,groupId)` I base on it :)

Reply:
yes it looks like an afterthought to the API. some knobs have an even crazier order (e.g. `options` or `files`) 🙄 that's why I couldn't just Search&Replace in #491.